beacon-chain/types/active_state.go

```diff
@@ -197,18 +197,32 @@ func (a *ActiveState) CalculateNewActiveState(block *Block, cState *Crystallized
 	}, newBlockVoteCache), nil
 }
 
+func (a *ActiveState) getBlockHash(block *Block, slot uint64) ([32]byte, error) {
+	sback := block.SlotNumber() - (params.CycleLength * 2)
+
+	if !(sback <= slot && slot < sback+(params.CycleLength*2)) {
+		//need to figure out what to do with this error, what happens then...?
```

Comment:
What's up with this comment? You're returning the error here.

Reply:
return the error is fine, we don't need this comment anymore

Reply:
the callers will take care of the error